java/src/apps/TrainCrew/TrainCrew_es.properties

```diff
@@ -1,3 +1,4 @@
+# java/src//apps/TrainCrew/TrainCrew_es.properties
```

Comment:
Several of these files have `//` in the file path in this line. Is this manually edited or could it be a bug in your new script? No need to fix in these files, but if there is a bug in the script, it would be good to fix the script.

Reply:
Thanks. Will fix.